bitnami/elasticsearch/templates/_helpers.tpl

```diff
@@ -126,9 +126,18 @@ Create a default fully qualified coordinating name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 */}}
 {{- define "elasticsearch.coordinating.fullname" -}}
+{{/* If Kibana is enabled. Evaluated in the Kibana subchart */}}
+{{- if .Values.enabled -}}
```

Comment:
What is this parameter?

Reply:
In the ES `values.yaml` we have a section with parameters affecting to the subchart:
```yaml
## Enable bundled Kibana
##
kibana:
  enabled: false
  elasticsearch:
    hosts:
      - '{{ include "elasticsearch.coordinating.fullname" . }}'
    port: 9200
```
Those parameters are available in the ES chart (main chart) as `.Values.kibana.XXX`, in the same way, those paremeters are used in the Kibana chart (subchart) as `.Values.XXX`.

In this section of the `helpers.tpl` I need to use both ways to determine if Kibana chart is used as part of an ES deployment. So the first `if` is evaluated in the subchart while the second `else if` is evaluated in the main chart; in both cases, the service name that returns the function is the same, that is the final goal

Reply:
I think this is very hacky. Why not using `global` to ensure both the chart and the subchart can access to these "global parameters"?